admin/c2cgeoportal_admin/__init__.py

```diff
@@ -1,13 +1,24 @@
+import c2cwsgiutils.pyramid
+# from c2cgeoportal_commons import models
 from pyramid.config import Configurator
+from c2cwsgiutils.health_check import HealthCheck
 
 
 def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application.
     """
     config = Configurator(settings=settings)
+    config.include(c2cwsgiutils.pyramid.includeme)
     config.include('pyramid_jinja2')
     config.include('c2cgeoform')
     config.include('c2cgeoportal_commons.models')
     config.include('.routes')
     config.scan()
+
+    health_check = HealthCheck(config)
+    # health_check.add_db_session_check(models.DBSession,
+    # at_least_one_model=models.User)
+    health_check.add_url_check('http://localhost/')
+    # health_check.add_alembic_check(models.DBSession, '/app/alembic.ini', 1)
```

Comment:
This section looks to be in development ...

Reply:
not in development, but for future reference

Reply:
ok